### PR TITLE
Fix formal verification CI

### PR DESCRIPTION
**Pull request by samuel40791765**
Resolve Formal Verification CI issues introduced by https://github.com/awslabs/aws-lc-verification/commit/e504944f94b6379eaf95f1146a168feb72ce844f

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and the ISC license.

### CODE REVIEW

File: tests/ci/run_formal_verification.sh [hunk 1]
```diff
@@ -7,7 +7,12 @@ cd ../
...
 
 rm -rf aws-lc-verification-build
-git clone --recurse-submodules https://github.com/awslabs/aws-lc-verification.git aws-lc-verification-build
+git clone https://github.com/awslabs/aws-lc-verification.git aws-lc-verification-build
+cd aws-lc-verification-build
+# We avoid pulling the saw-script submodule since the repo will take forever to clone.
+git submodule update —init
+pushd Coq/fiat-crypto; git submodule update --init --recursive; popd
+
 cd aws-lc-verification-build
```
Comment (pennyannn) [inline tests/ci/run_formal_verification.sh:16]: This line Could be removed

Comment (samuel40791765) [reply tests/ci/run_formal_verification.sh]: Didn't notice that, thanks!

File: tests/ci/run_formal_verification.sh [hunk 2]
```diff
@@ -7,8 +7,13 @@ cd ../
 ROOT=$(pwd)
 
 rm -rf aws-lc-verification-build
-git clone --recurse-submodules https://github.com/awslabs/aws-lc-verification.git aws-lc-verification-build
+git clone https://github.com/awslabs/aws-lc-verification.git aws-lc-verification-build
+
 cd aws-lc-verification-build
+# We avoid pulling the saw-script submodule since the repo will take forever to clone.
+git submodule update —init
```
Comment (pennyannn) [inline tests/ci/run_formal_verification.sh:14]: `—init` to `--init`. (Though I'm not sure what is going on with the dashes there)

**Review by pennyannn — APPROVED** [review]
LGTM

**Review by skmcgrail — APPROVED** [review]

**Review by nebeid — APPROVED** [review]

**Pull request merged**